src/VirtualClient/VirtualClient.Main/profiles/MONITORS-DEFAULT.json

```diff
@@ -37,37 +37,6 @@
         "MonitorFrequency": "$.Parameters.MonitorFrequency",
         "MonitorWarmupPeriod": "$.Parameters.MonitorWarmupPeriod"
       }
```

Comment:
Thank you for spotting this. I remember I did it but must have lost it in some commits.